.github/workflows/clp-docs.yaml

```
@@ -39,3 +39,17 @@ jobs:
       - name: "Build docs"
         shell: "bash"
         run: "task docs:site"
+
+      # Upload the built docs so we can download and deploy them from y-scope/yscope-docs
+      - if: >-
+          contains(fromJSON('["push", "workflow_dispatch"]'), github.event_name)
+          && ('refs/heads/main' == github.ref || startsWith(github.ref, 'refs/tags/v'))
+          && 'ubuntu-latest' == matrix.os
+        uses: "actions/upload-artifact@v4"
+        with:
+          name: "docs-html"
+          path: "build/docs/html"
+          if-no-files-found: "error"
+
+          # Retain the artifact for a week in case there was a deployment issue
+          retention-days: "7"
```

Though it appears that the input parser of the action does parse the field as a string: https://github.com/actions/upload-artifact/blob/4cec3d8aa04e39d1a68397de0c4cd6fb9dce8ec1/src/upload/input-helper.ts#L35

The type of the input is defined as `number` at https://github.com/actions/upload-artifact/blob/4cec3d8aa04e39d1a68397de0c4cd6fb9dce8ec1/src/upload/upload-inputs.ts#L22

If always putting values as quotes is not a convention that we want to follow, i feel it's better to put `7` instead of `"7"`.
```suggestion
          retention-days: 7
```

---

My mistake.

---

kk